subjects/onlyf/README.md

```diff
@@ -0,0 +1,16 @@
+## onlyf
+
+### Instructions
+
+Write a program that displays an `f` character on the standard output. (and nothing else)
```

Comment:
Instead of "Write a program that displays an `f` character on the standard output. (and nothing else)" I would suggest "Write a program that displays an `f` character on the standard output, and nothing else."

Reply:
fixed